src/rust/engine/fs/brfs/src/main.rs

```diff
@@ -763,14 +763,22 @@ async fn main() {
 
   let runtime = task_executor::Executor::new();
 
+  let tls_config = match tls::Config::new(root_ca_certs, None) {
+    Ok(tls_config) => tls_config,
```

Comment:
It would be more consistent with other code here to use `expect()`.

Reply:
Ten lines up is another error + exit, which is what I mirrored.